content/cloud/noops-sql-db.md

```diff
@@ -34,69 +36,73 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 * Your NoOps SQL Database can hold up 1 GB of data
 
 *Service Limitations*
-* The `--sqlite` flag not supported on `spin cloud deploy`
+* The `--sqlite` flag is not supported on `spin cloud deploy`
 * Fermyon **Cloud** only supports databases named "default" at this time
 
-## How To:
+## Creating a New Spin Application
 
-### Prepare a Database
+If you already have a Spin application, you can skip this step. If you do not have a Spin application locally you will need to [create one](/spin/quickstart#create-your-first-application).
 
-You can use the `spin cloud sqlite execute` command to check if a table needs to be created, and if so create one: 
+## Grant SQLite Permission
 
-```bash
-$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
+To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest](/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components) (the `spin.toml` file). For example:
+
+```
+[component]
+sqlite_databases = ["default"]
 ```
 
->> If you need to look up the name of your database, use the `spin cloud sqlite list` command
+The new database file (`sqlite_db.db`) is created in your application's `.spin` folder when you deploy this Spin application. The database persists across Spin application invocations and updates. The database is empty and can optionally have tables and data added using the following step.
 
-### Manage the Database Lifecycle
+## Managing You Local Spin Database Schema
```

Comment:
```suggestion
## Managing Your Local Spin Database Schema
```

Reply:
Thanks @itowlson 
This is in the new PR.